### ISSUE TITLE
Peer of eslint@^2.0.0 required but none was installed

### ISSUE THREAD

**Issue by hessler** [lo1tuma/eslint-plugin-mocha#122]
Noticing the following error in terminal, wondering if an update is needed for this plugin? Please advise. Thanks!

```
npm WARN eslint-plugin-mocha@2.2.0 requires a peer of eslint@^2.0.0 but none was installed.
```

FWIW, I have also ensured that the most up-to-date versions of both packages are installed.

**Comment by lo1tuma** [comment]
This warning actually means that you haven’t installed any version of eslint.
